Packs/GSuiteSecurityAlertCenter/Integrations/GSuiteSecurityAlertCenter/GSuiteSecurityAlertCenter.yml

```diff
@@ -0,0 +1,805 @@
+      description: The time the metadata was last updated.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.metadata.severity
+      description: The severity value of the alert.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.metadata.etag
+      description: Etag is used for optimistic concurrency control as a way to help
+        prevent simultaneous updates of an alert metadata from overwriting each other.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.@type
+      description: Alert data of alert types.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.takeoutRequestId
+      description: The takeout request ID.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.email
+      description: "The email of the associated event. \nDomain wide takeout initiated\
+        \ implies to the email of the admin who initiated the takeout.\nGoogle identity\
+        \ implies to the email of the user that the event belongs to.\nMobile device\
+        \ management/State Sponsored Attack implies to the email of the user the alert\
+        \ was created for."
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.domainId.customerPrimaryDomain
+      description: The primary domain for the customer.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.maliciousEntity.entity.emailAddress
+      description: Email address of the user.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.maliciousEntity.entity.displayName
+      description: Display name of the user.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.maliciousEntity.fromHeader
+      description: The sender email address.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.maliciousEntity.displayName
+      description: The header from display name.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.messages.messageId
+      description: The message ID.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.messages.md5HashMessageBody
+      description: The hash of the message body text.
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.messages.messageBodySnippet
+      description: The snippet of the message body text (only available for reported
+        emails).
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.messages.md5HashSubject
+      description: The MD5 Hash of email's subject (only available for reported emails).
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.messages.subjectText
+      description: The email subject text (only available for reported emails).
+      type: String
+    - contextPath: GSuiteSecurityAlert.Alert.data.messages.attachmentsSha256Hash
+      description: The SHA256 hash of email's attachment and all MIME parts.
+      type: Unknown
```

Comment:
Please change the type

Reply:
Array is returned as part of  `attachmentsSha256Hash` field that's why we have kept type as Unknown

Reply:
OK, so please mention in the description that this is an array

Reply:
Will change as required.